script/acceptancetest_environments/keystonev2-lbaasv1.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
```

Comment:
Did you mean for this to be committed, and, if so, can you comment on what it does? Something like `# Installs keylogger` would suffice.